docs/index.rst

```diff
@@ -9,6 +9,7 @@ Manticore is a prototyping tool for dynamic binary analysis, with support for sy
 
    api
    models
+   internal
```

Comment:
i'm not sure we should have internal classes documented on readthedocs, I sort of feel like it will only encourage people to use these classes/methods that they shouldn't. the main people that should be reading those internal docs anyway are developers, who should be able to read in the source in their editor.